mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -91,7 +91,7 @@ impl Bridge {
         pin_mut!(local_pump, remote_pump);
 
         debug!(
-            "Starting pumps for {} bridge...",
```

Comment:
The pattern I have observed is that it should only be capital if more than one sentence.